src/objc/nntp/MCONNTPSendOperation.h

```diff
@@ -0,0 +1,41 @@
+//
+//  MCONNTPSendOperation.h
+//  mailcore2
+//
+//  Created by Daryle Walker on 2/21/16.
+//  Copyright © 2016 MailCore. All rights reserved.
+//
+
+#ifndef MAILCORE_MCONNTPSENDOPERATION_H
+
+#define MAILCORE_MCONNTPSENDOPERATION_H
+
+#import <Foundation/Foundation.h>
+#import <MailCore/MCONNTPOperation.h>
+
+/** Send a message from NNTP3 */
+
+typedef void (^MCONNTPOperationProgressBlock)(unsigned int current, unsigned int maximum);
```

Comment:
Please move it to MCONNTPOperation since the same is defined in NNTPFetchArticleOperation.